index.html

```diff
@@ -192,21 +215,21 @@ <h2 class="component-name">Avatar</h2>
               <article class="component component-list component-center">
```

Comment:
I observed that you have followed semantic html ,it is really good to read code which is semantically written

Reply:
Thank you @amansethi00 for the review and suggestions.
Move box-shadow and linear-gradient to variables, use variable instead of hardcoded color value.